propka/lib.py

```diff
@@ -272,3 +287,31 @@ def writeFile(filename, lines):
     f.close()
 
 
+def info(*args, **kargs):
+    """Log a message. Level defaults to INFO unless overridden."""
+    level = kargs.pop("level",logging.INFO)
+    for l in _sprint(*args, **kargs):
+        logger.log(level,l)
+
+
+def info_debug(*args, **kargs):
+    """Log a message on the DEBUG level."""
+    info(*args, **kargs, level=logging.DEBUG)
+
+
+def info_warning(*args, **kargs):
+    """Log a WARN message"""
+    for l in _sprint(*args, **kargs):
+        logger.warning("Warning: "+l)
+
+def _sprint(*args, **kargs):
```

Comment:
This feels hacky.   Could we have info(), warning(), and debug() functions accept just *args that we could join together, without printing to QString?   Why are **kwargs needed?

Alternatively we could update the "print" statements in the codebase to call these functions with a single string (some would need to be updated to format multiple args into a single string).